scripts/iam-tools/aws_creds_check.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
```

Comment:
There should be a comment block at the top detailing what this script's purpose is so that future readers don't have to read the whole script just to get an idea of what it's supposed to be doing and why. It doesn't need to be a novel. Just a couple sentences should be fine.